tensorflow/lite/micro/examples/micro_speech/CEVA_BlueVox/main_functions_cpp.cc

```diff
@@ -0,0 +1,306 @@
+
+
```

Comment:
Can you add the standard TensorFlow copyright notice here?
The file should also be formatted using clang-format -style=google

Reply:
How do I do that I'm new at this